### DISCUSSION TITLE
Turbopack not working in versions newer than 15.0.0-canary.91

### DISCUSSION THREAD

**Discussion by EliasVal** [vercel/next.js#68657] [Help]
### Summary

Hello! I kept my app on 15.0.0-canary.91 since Turbopack is stuck on `Compiling / ...` in any version newer than that...
I have no idea why that's happening and any help would be appreciated :)

### Additional information

```js
Operating System:
  Platform: win32
  Arch: x64
  Version: Windows 11 Pro
  Available memory (MB): 32457
  Available CPU cores: 24
Binaries:
  Node: 20.16.0
  npm: N/A
  Yarn: N/A
  pnpm: N/A
Relevant Packages:
  next: 15.0.0-canary.108 // Latest available version is detected (15.0.0-canary.108). 
  eslint-config-next: 15.0.0-canary.102
  react: 19.0.0-rc-8269d55d-20240802
  react-dom: 19.0.0-rc-8269d55d-20240802
  typescript: 5.5.4
Next.js Config:
  output: N/A
```

**Comment by EliasVal** [comment]
Additional context: The project is a part of a monorepo

**Comment by fily1212** [comment]
I have the same problem, with last Next.js project. I'm in Ubuntu 24.04, Node.js with 22.10, I create with npx create-next-app@latest , then npm run dev and it's stuck in compiling with turbopack...

**Reply by EliasVal** [reply]
Hey! I've fixed this issue since. IIRC, the solution was, in the context of a Monorepo at least, to get rid of ALL `package-lock.json`, `pnpm-lock.yaml`, etc that you have, delete all `node_modules` in the monorepo.
Next, check the root `package.json` to see if it has Next.js specified there, if so, remove it. Next.js should only be specified within the workspaces and not the monorepo.

Afterwards, run the install command with your package manager.

**Reply by EliasVal** [reply]
If you're still having trouble, make sure your Next.js and React versions are up-to-date. For Next.js `15.0.2-canary.10`, the corresponding React and React-DOM version is `19.0.0-rc-1631855f-20241023`
